src/main/java/com/oculusvr/capi/TrackingState.java

```diff
@@ -13,6 +13,11 @@
   public PoseStatef[] HandPoses = new PoseStatef[2];
```

Comment:
this doesn't match the ovr struct in 1.3.... 

`CameraPose`, `LeveledCameraPose`, `RawSensorData`, `LastCameraFrameCounter` and `padding` all need to be removed
`StatusFlags` and `HandStatusFlags` are in the wrong location relative to the structure
`CalibratedOrigin` is missing